apps/authentication/models.py

```diff
@@ -491,7 +491,7 @@ class OnlineGroup(ObjectPermissionModel, models.Model):
         on_delete=models.CASCADE,
     )
 
-    """ The short name of a group, eg. HS or Dotkom """
+    """ The short name of a group, eg. HS or dotkom """
```

Comment:
This spelling is unrelated to by original issue, but I have inquired {p,P}rokom about what the propper spelling for committee-short-names should be